docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -220,6 +220,8 @@ In {product-kafka}, you can create Access Control Lists (ACLs) in your Kafka ins
 
 .. Click *Manage access*.
 .. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
++
+If you don't see users in the drop-down list, ask your organization administrator xref:{base-url}{access-mgmt-url-kafka}#proc-user-account-access-managing-access[to grant access to view other user accounts].
```

Comment:
I think the IBM SG prefers to avoid linking text as text and instead use a bona fide cross-ref, like "ask your organization administrator to grant access to view other user accounts. For more information, see xref:....[Using RBAC to ...]."  I'm not much of a fan of this rule, but we've tended to it mostly till now so just thought might be good for consistency. Up to you.

Reply:
The _For more information..._ can interrupt the flow a bit. And it's more text. :-). But you're closer to the docs, so we'll go with consistency. Updated.